src/legacy/core_plugins/kibana/public/dashboard/panel/_dashboard_panel.scss

```diff
@@ -55,98 +73,110 @@
 // Adjust borders/etc... for non-spaced out and expanded panels
 .dshLayout-withoutMargins,
 .dshDashboardGrid__item--expanded {
-  // Remove border color unless in editing mode
-  .dshPanel:not(.dshPanel--editing) {
-    border-color: transparent;
-  }
-
   .dshPanel {
     box-shadow: none;
     border-radius: 0;
   }
 }
 
+// Remove border color unless in editing mode
+.dshLayout-withoutMargins:not(.dshLayout--editing),
+.dshDashboardGrid__item--expanded {
+  .dshPanel {
+    border-color: transparent;
+  }
+}
+
 // HEADER
 
 .dshPanel__header {
-  padding: $euiSizeS;
   flex: 0 0 auto;
-  white-space: nowrap;
   display: flex;
-  border: none;
-  transition: background-color $euiAnimSpeedFast $euiAnimSlightResistance;
+  // ensure menu button is on the right even if the title doesn't exist
+  justify-content: flex-end;
 }
 
 .dshPanel__title {
-  font-size: $euiFontSizeS;
-  color: $euiColorDarkShade;
-  transition: color $euiAnimSpeedFast $euiAnimSlightResistance;
-  flex: 1 1 auto;
   @include euiTextTruncate;
+  @include euiTitle('xxxs');
+  line-height: 1.5;
+  flex-grow: 1;
 
-  // flexbox fix for IE10
-  // http://stackoverflow.com/questions/22008135/internet-explorer-10-does-not-apply-flexbox-on-inline-elements
-  display: inline-block;
+  &:not(:empty) {
+    padding: ($euiSizeXS * 1.5) $euiSizeS 0;
+  }
 }
 
-.dshPanel--editing:hover {
-  .dshPanel__header {
-    background-color: rgba($euiColorFullShade,.05) !important;
-    cursor: move;
+.dshLayout--editing {
+  .dshPanel__dragger {
+    transition: background-color $euiAnimSpeedFast $euiAnimSlightResistance;
   }
 
-  .dshPanel__title:hover {
-    color: $euiColorPrimary;
+  .dshPanel__dragger:hover {
+    background-color: transparentize($euiColorWarning, .9);
```

Comment:
Might want to adjust this a bit for dark mode

![image](https://user-images.githubusercontent.com/324519/52189007-d328ae80-27ea-11e9-9ca8-aefa2d6ededb.png)